ACL/acl-schema.sql

```diff
@@ -0,0 +1,97 @@
+create database if not exists acl;
+use acl;
+ 
+#Hence user type is new user(root user) or administrater
+CREATE TABLE IF NOT EXISTS userType(
+    type_id int,
```

Comment:
make it auto_increment.

Reply:
file should follow the correct path. it should be present in `<project-name>/resource/sql` directory